### DISCUSSION TITLE
Robot startup issue

### DISCUSSION THREAD

**Discussion by myszzka96** [HumanSupportRobot/QandA#57] [Q&A]
Dear admin,

We have noticed that every time we turn on the robot while the head or the wrist are not in a centered position (or at least not laying at the extreme positions in the movement range), the robot produces an error message and won't start up properly. Then the head and wrist positions have to be corrected and the robot restarted. After that, it usually starts up properly. I am unsure what is this exactly related to but would appreciate some insight and possible solution.

Thank you!

**Comment by ayuguchi** [comment]
Although I'm only a user, my solution is that run `whole_body.move_to_neutral()` on the HSRB interface to move the neutral position before turning off an HSR.
